modules/migration/view/list.php

```diff
@@ -0,0 +1,78 @@
+<html>
+<head>
+<style type='text/css'>
+  * {
+    font-size: 12px;
+  }
+</style>
+</head>
+<body>
+<?php
+if ($_lib['sess']->get_person('AccessLevel') < 4) {
+  header('Location: ' . $_lib['sess']->dispatchs . "t=lodo.main");
+  die();
+}
+
+includelogic("migration/migration");
+global $_lib;
+
+$migration_system = new migration_system();
+
+$db_name_restriction = (isset($_REQUEST['db_name'])) ? $_REQUEST['db_name'] : NULL;
```

Comment:
Maybe do not even get the unwanted migrations instead of only not printing them.